services/services.go

```diff
@@ -0,0 +1,68 @@
+package services
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/api/types"
+	"github.com/rancher/rke/hosts"
+	"golang.org/x/net/context"
+)
+
+type Container struct {
```

Comment:
Why do we need this wrapper struct?

Reply:
because we need a container struct with name `yaml: services` to contain each service type